tsconfig.json

```diff
@@ -7,9 +7,7 @@
     "outDir": "lib",
     "removeComments": true,
     "sourceMap": true,
-    "target": "ES6"
+    "target": "es2015"
```

Comment:
Node 6 supports 99% of ES2015, and we run into issues with `Map` and `Promise` if we don't change the target.

Reply:
Isn't `es6` and `es2015` equivalent?

You might just have merged and not noticed it's already `es6`? I'm not sure how these two are different.